test/test-utils.ts

```diff
@@ -1,36 +1,16 @@
-import WebSocket from 'ws';
-import nock from 'nock';
-import crypto from 'crypto';
-import fs from 'fs';
-import path from 'path';
+
```

Comment:
I created this file for unit tests of the SDK client. Most of these tests came from `client.integration.test.ts` with some minor refactors. I've been wanting to do that since not all sdk tests need to be integration. This keeps things simple for simpler tests. It also saves on avg 10+ seconds because the integration tests take longer to run.